CMakeLists.txt

```diff
@@ -87,7 +118,7 @@ if (MSVC)
     list(APPEND CMAKE_REQUIRED_FLAGS /wd4996)
 endif()
 
-if (NOT DISABLE_STATIC_FPIC)
+if (NOT DISABLE_STATIC_FPIC AND NOT AMIGA)
```

Comment:
If `CMAKE_POSITION_INDEPENDENT_CODE` doesn't work, how about setting `DISABLE_STATIC_FPIC` above, where you disable various other things?

Reply:
Done